### DISCUSSION TITLE
Remove ICC profile without reencoding the image (non destructive)

### DISCUSSION THREAD

**Discussion by lanwin** [SixLabors/ImageSharp#1516] [Q&A] [answered]
Hi, is there a way to remove the ICC Profile from an image without encoding it again....non destructive?

```chsarp
using(var image = Image.Load(stream, out var format))
{
    image.MetaData.IccProfile = null;
    image.Save(memory, format);
}
```

This works well, but it also reencodes the image again.

**Comment by brianpopow** [comment, marked as answer]
No that is not possible, it needs to be re-encoded again.
